packages/chopsticks/src/context.ts

```diff
@@ -34,5 +35,15 @@ export const setupContext = async (argv: Config, overrideParent = false) => {
   await overrideStorage(chain, argv['import-storage'], at)
   await overrideWasm(chain, argv['wasm-override'], at)
 
+  // load blocks from db
```

Comment:
is this the best place to load blocks? may need to consider do this before or after overrideStorage

Reply:
Yeah I think load db before override so the overrides actually works.